django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -34,6 +34,16 @@ def as_sql(self, connection, lookup, template_params, sql_params):
         return super(SpatiaLiteDistanceOperator, self).as_sql(connection, lookup, template_params, sql_params)
 
 
+class SpatialiteNullCheckOperator(SpatialOperator):
+    def as_sql(self, connection, lookup, template_params, sql_params):
+        if 'rhs' in template_params:
+            sql_template = '%(func)s(%(lhs)s, %(rhs)s) > 0'
+        else:
+            sql_template = '%(func)s(%(lhs)s) > 0'
+        template_params.update({'op': self.op, 'func': self.func})
+        return sql_template % template_params, sql_params
```

Comment:
By the way, [looking up `SpatialOperator`'s implementation](https://github.com/django/django/blob/31a2af1c0101081a3950dab0e66fa41bbbf6d34f/django/contrib/gis/db/backends/utils.py#L7-L27) I think you could probably go away by simply overriding `default_template` instead.

```python
class SpatialiteNullCheckOperator(SpatialOperator):
    @property
    def default_template(self):
        return '%s > 0' % super().default_template
```